docs/src/examples/components/Label/Variations/LabelExampleImage.shorthand.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Label } from '@stardust-ui/react'
+
+const LabelExampleImageShorthand = () => (
+  <Label
+    circular
+    icon="close"
+    iconPosition="end"
```

Comment:
What happens if you add an icon, but not `iconPosition="end"`?  If both the `icon` and the `image` are added to the `start` position by default, which I think makes sense, then perhaps we call these `media` or `startMedia` and `endMedia` instead?

Trying to think about how this aligns to the similar case of the ListItem.  In that case, you can add `media` and `endMedia`.  The layout of both a ListItem and a Label are idential in this regard.  The start and end media pieces collapse their width while the main area expands to fill the space.

The flip side of this argument is that perhaps it might be more common and more intuitive to add an image to the "start" side by default but maybe icons are most intuitive on the "end" side by default.  Therefore, a position prop is introduced to allow positioning them on the less common/intuitive side when needed.

Feedback?

Reply:
@levithomason I already created PR for the ItemLayout component, and yes, after working with it I tried to re-implement the Label with the startMedia and endMedia and it it much cleaner. With that we would get rid of the icon, iconPosition and image props and have more abstract way of defining the start and end media in the Label, as well as other components. My proposal is, to wait for the ItemLayout component PR to be merged and immediately after it, I will publish the PR for the Label containing the start and end media, as I already have it ready on a separate branch.